src/main/kotlin/no/nav/medlemskap/common/ExceptionHandler.kt

```diff
@@ -9,12 +9,19 @@ import io.ktor.http.HttpStatusCode
 import io.ktor.request.uri
 import io.ktor.response.respond
 import mu.KotlinLogging
+import no.nav.medlemskap.common.exceptions.IdenterIkkeFunnet
 import no.nav.medlemskap.common.exceptions.PersonIkkeFunnet
 import no.nav.medlemskap.common.exceptions.Sikkerhetsbegrensing
 
 private val logger = KotlinLogging.logger { }
 
 fun StatusPages.Configuration.exceptionHandler() {
+    exception<IdenterIkkeFunnet> { cause ->
+        call.logErrorAndRespond(cause, HttpStatusCode.NotFound) {
+            "Fant ingen aktør-id for fødselsnummer"
```

Comment:
Når jeg titter i ExceptionHandler, så lurer jeg litt på om vi kunne optimalisert det litt, f.eks ved å ha feilmeldingen ("Fant ingen aktør-id for fødselsnummer" etc) i exceptionen, og ikke i handleren, og kanskje redusert antall exceptions vi bruker, så handleren ble mer generisk, og ikke måtte tilpasses hver feilsituasjon.. Men det er ikke så gjennomtygd, og det kan evt. gjøres seinere.. :)